shared/editor/nodes/CodeFence.ts

```diff
@@ -134,12 +140,56 @@ export default class CodeFence extends Node {
           option.selected = node.attrs.language === value;
           select.appendChild(option);
         });
+        const codeClass = !node.attrs.diagram ? "editor-visible" : "";
+
+        const diagram = document.createElement("div");
+        if (node.attrs.language === "mermaidjs") {
+          const id = Math.round(Math.random() * 100000);
```

Comment:
This is going to generate a new `id` every time it's rendered – that seems, bad?